### PR TITLE
based on worker/nfs PR, refactored the code to load config and create clusterID

### PR DESCRIPTION
**Pull request by YinYangOfDao**

### CODE REVIEW

File: src/ClusterBootstrap/deploy.py [hunk 1]
```diff
@@ -284,82 +284,6 @@ def create_cluster_id():
         config["clusterId"] = utils.get_cluster_ID_from_file()
         print "Cluster ID is " + config["clusterId"]
 
-def add_acs_config(command):
```
Comment (Anbang-Hu) [inline src/ClusterBootstrap/deploy.py]: Are we sure that this method is no longer needed?

File: src/utils/DockerUtils.py [hunk 2]
```diff
@@ -219,7 +219,7 @@ def config_dockers(rootdir, dockerprefix, dockertag, verbose, config):
         system_docker_dic = config["dockers"]["system"]
         docker_list = get_docker_list(rootdir, dockerprefix, dockertag, None, verbose )
         # Populate system dockers 
-        for assemblename, tuple in docker_list.iteritems():
+        for assemblename, tuple in docker_list.items():
```
Comment (Anbang-Hu) [inline src/utils/DockerUtils.py]: change `tuple` to `tupl` as well

File: src/docker-images/cloudinit/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# This folder builds the webui docker.
```
Comment (Anbang-Hu) [inline src/docker-images/cloudinit/README.md]: not for webui

File: src/ClusterBootstrap/deploy.py (hunk 1 above)
Comment (YinYangOfDao) [reply src/ClusterBootstrap/deploy.py]: yes, no longer needed. not only this one, but also others like hdfs glusterdfs etc.

File: src/ClusterBootstrap/cloud_init_aztools.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+	scaledown shutdown a particular VM.
+	list list VMs.
+	interconnect create network links among VMs
+	genconfig Generate configuration files to describe the static status for Azure VM cluster.
+	''') )
+	parser.add_argument("command", help="See above for the list of valid command")
+	parser.add_argument('nargs', nargs=argparse.REMAINDER, help="Additional command argument")
+	output_file_name="azcluster.yaml"
+	parser.add_argument("--complementary",
+		help="Specify complementary file name the number of infra nodes, default = ",
```
Comment (Anbang-Hu) [inline src/ClusterBootstrap/cloud_init_aztools.py]: Remove `default =` in `help`

**Pull request merged**